src/sass/components/ion-content.scss

```diff
@@ -14,4 +14,8 @@ ion-content {
   &.padded-bottom-cta-with-summary {
     bottom: 134px;
   }
+
+  &.scroll-content {
+    margin-top: -2px;
```

Comment:
By default it is -1px, I adjusted with -2px to match perfectly with the orange bar from at top bar.

Reply:
I don't understand this change. The header bar is 44px high, and the content is absolutely positioned at 44px from the top.  Seems to me  like a margin of 0 would be about right (perhaps they made it -1px to enforce scrolling?) What's the need you had that caused you to make this change?